app/_src/guides/rules-kv.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Configuring inbound traffic with rules API
+---
+
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+{% assign kuma-system = site.mesh_namespace | default: "kuma-system" %}
+{% assign kuma-control-plane = kuma | append: "-control-plane" %}
+
+Using `Rules API` combined with targetRef `Dataplane` kind in {{site.mesh_product_name}} you can easily apply configuration to [data plane](/docs/{{ page.release }}/introduction/concepts/#data-plane)
+[inbounds](/docs/{{ page.release }}/introduction/concepts/#inbound). This guide will show you how to configure [MeshTimeout](/docs/{{ page.release }}/policies/meshtimeout) policy on data plane inbound and explain how to utilize this API.
```

Comment:
I remembered that somewhere we had explained the concept of `Rules API`, can we supplement a {% tip %} here to describe it or linked URL? Since this is a `guide` and user may don't have too much knowledge about our terms. WDYT?

Reply:
This guide purpose is to explain what rules api is, so I don't think we need to duplicate it. You either read about it in theory or do a guide and have all the needed knowledge